spyder/utils/qthelpers.py

```diff
@@ -502,7 +502,42 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
-    
+
+def create_control_layout(controls, main_widget):

```

Comment:
I'm not sure what if this is the best way to do this:
- It take in account plugins with one controls line (for example findinfiles have two lines), if we are going to standardize how controls are created I think that should be for all plugins
- I think the name isn't the best, sometimes it creates the main layout of the plugin, and sometimes it's the controls layout

A more general solution (out of the scope of this PR) thinking also about third party plugins, could be abstract the creation of the Main layout in the SpyderPlugin class, a plugin will define: a main widget, a controls_bar, and corner_actions; the plugins will lose flexibility but we could guaranty that the all look the same.